orchestrator/ethereum_gravity/src/utils.rs

```diff
@@ -149,28 +149,26 @@ pub async fn get_gravity_id<S: Signer + 'static>(
                     if gravity_id != contract_id_value {
                         error!("Contract gravity id does not match with the chain gravity id");
                         return Err(GravityError::GravityContractError(format!(
-                            "Gravity contract id {} does not match with chain gravity id {}",
-                            gravity_id,
-                            contract_id_value,
-                        )))
+                            "Gravity contract id {gravity_id} does not match with chain gravity id {contract_id_value}"
+                        )));
                     }
 
-                    info!("Gravity contract id {} match with chain gravity id {}", gravity_id, contract_id_value);
+                    info!(
+                        "Gravity contract id {} match with chain gravity id {}",
```

Comment:
I saw a lot of the previous changes tried to switch to the "implicit captures", so this could potentially use it as well (`contract id {gravity_id}` ....`) to keep it consistent?